x-pack/solutions/security/plugins/security_solution/public/flyout/document_details/shared/hooks/use_graph_preview.ts

```diff
@@ -77,6 +90,7 @@ export const useGraphPreview = ({
     actorIds.length > 0 &&
     eventIds.length > 0 &&
     targetIds.length > 0;
+  const { isAlert } = useBasicDataFromDetailsData(dataFormattedForFieldBrowser);
```

Comment:
sorry for my late reply @kfirpeled, this looks good, thanks for making the change! I thought this logic could actually have been done within the components directly, instead of passing `dataFormattedForFieldBrowser` to this hook then retrieve the `isAlert` from here, but this works as well!